src/expr/src/vector_op/arithmetic_op.rs

```diff
@@ -201,6 +201,22 @@ where
     interval_int_mul::<T2, T1, T3>(r, l)
 }
 
+#[inline(always)]
+pub fn interval_float_div<T1, T2, T3>(l: IntervalUnit, r: T2) -> Result<IntervalUnit>
+where
+    T2: TryInto<OrderedF64> + Debug,
+{
+    l.div_float(r).ok_or(ExprError::NumericOutOfRange)
```

Comment:
Actually PG will return a "division by zero" error, but this is acceptable to me.